src/create_input.py

```diff
@@ -20,6 +20,7 @@
 import itertools
 
 p835_personalized = 'pp835'
+p804_conference = 'p804_conference'

```

Comment:
It seems that sole purpose of adding a new test type is a change to a single paragraph in the template. There should be more efficient way to achieve the same result.

If we still need a new method, then the methods needs to be added also to `results_parser.py`.